packages/frontend/editor-ui/src/features/logs/composables/useChatState.ts

```diff
@@ -154,6 +165,37 @@ export function useChatState(isReadOnly: boolean): ChatState {
 		const response = await runWorkflow(runWorkflowOptions);
 
 		if (response) {
+			const wsProtocol = rootStore.urlBaseEditor.startsWith('https') ? 'wss' : 'ws';
+			const wsUrl = rootStore.urlBaseEditor.replace(/^https?/, wsProtocol).replace(/\/$/, '');
+			ws.value = new WebSocket(
+				`${wsUrl}/chat?sessionId=${currentSessionId.value}&executionId=${response?.executionId}`,
```

Comment:
Could we share the setup of this WebSocket with the one in the chat? Maybe in a composable exposed from the chat package?